eng/testing/WasmRunnerTemplate.sh

```diff
@@ -8,9 +8,9 @@ cd $EXECUTION_DIR
 XHARNESS_OUT="$EXECUTION_DIR/xharness-output"
 
 dotnet xharness wasm test --engine=$JAVASCRIPT_ENGINE \
-    --js-file=runtime.js \
+    --js-file=runtime.js -v \
     --output-directory=$XHARNESS_OUT \
-    -- --enable-gc --run WasmTestRunner.dll ${@:2}
+    -- --enable-gc --setenv=MONO_LOG_MASK= --run WasmTestRunner.dll ${@:2}
```

Comment:
I'd prefer having an `--enable-gc-logging` switch that turns on this logging in driver.c and have it disabled by default, rather than this.